### ISSUE TITLE
An Idea for a unit converter

### ISSUE THREAD

**Issue by bb1a2** [SimpleMobileTools/General-Discussion#74]
hide categories and units, reversible. autofocus on edit
I was watching a TEDx talk some time ago where an app developer tells[(https://www.youtube.com/watch?v=nmZR_JsLCfg) how he learned to code and discussed his transportation [app](https://itunes.apple.com/za/app/gaurider/id628358170).

I liked the intuitive way he made his app, and I found almost all unit converters required too many steps to get the results. For example, after you open an [app](https://play.google.com/store/apps/details?id=com.physphil.android.unitconverterultimate):

- choose category
- choose 'from' unit
- choose 'to' unit
- tap edit field
- type value

Some unit converters try to do away with selecting a 'to' unit by having a field next to all units and type in one field to get results in all the others.

An idea similar to the one used by the transportation app is to have units in a circle or another arrangement where you can swipe from one unit to another:

**Comment by tibbi** [comment]
It might be added someday in the far future. Maybe as part of the calculator, maybe not. Dont expect having it anytime soon though.

**Comment by tibbi** [comment]
not really. There are a lot of unit converters out there, just search for some.

**Comment by bb1a2** [comment]
I did :). T tried dozens of them. I didn't find FOSS/good ones with that feature.

Thank you for your prompt responses by the way.
